book/src/themes.md

```diff
@@ -295,6 +295,7 @@ These scopes are used for theming the editor interface:
 | `ui.text.inactive`                | Same as `ui.text` but when the text is inactive (e.g. suggestions)                             |
 | `ui.text.info`                    | The key: command text in `ui.popup.info` boxes                                                 |
 | `ui.virtual.ruler`                | Ruler columns (see the [`editor.rulers` config][editor-section])                               |
+| `ui.virtual.diagnostics`    | Default style for inline diagnostics lines (notably control the background)                    |
```

Comment:
I actually don't allow theming the background right now. The docs are the only thing I didn't update yet

Reply:
![Screenshot 2023-03-26 at 19 53 44](https://user-images.githubusercontent.com/7951708/227794638-afe7a3c4-34ca-4587-877d-f4e13d0a1f20.png)

Without a background to errors it's pretty unreadable

![Screenshot 2023-03-26 at 19 57 18](https://user-images.githubusercontent.com/7951708/227794805-8034546a-5f1e-4d13-a457-10f973833ce8.png)

With one it's a little readable but still much less pleasant to use than what I had in my PR, I want to known when lines are virtual.

What's more, by setting a background for errors, now my status line is ugly:

![Screenshot 2023-03-26 at 19 58 39](https://user-images.githubusercontent.com/7951708/227794886-dbc0dec8-a8e4-408b-86c1-c4b1e6b3f727.png)

Reply:
Virtual lines should always have the option to be themed entirely differently, right now this is a readability nightmare, either I break my status line or the document display 😞 plus I can't know at a glance if missing line numbers are because of virtual text of softwrapping or because `rustfmt` crashed